examples/examples.ipynb

```diff
@@ -896,7 +896,7 @@
     "from sklearn.naive_bayes import GaussianNB\n",
     "from sklearn.linear_model import LogisticRegression \n",
     "from sklearn.ensemble import RandomForestClassifier\n",
-    "from sklearn.cross_validation import train_test_split\n",
+    "from sklearn.model_selection import train_test_split\n",
```

Comment:
Updated the import statement since scikit-learn has deprecated the `cross_validation` module in favor of `model_selection`.

Reply:
Thanks for implementing this fix @Kautumn06!